### DISCUSSION TITLE
Keybind to display all Keys

### DISCUSSION THREAD

**Discussion by notwidow** [qtile/qtile#4031] [Q&A]
Hi, Is there a keybind which displays all other keybindings? 
and does qtile tells us if we assign a keybind that is already taken?

**Comment by m-col** [comment]
> Hi, Is there a keybind which displays all other keybindings?

You may be interested in this discussion: https://github.com/qtile/qtile/issues/1329

>  and does qtile tells us if we assign a keybind that is already taken?

No, which is pretty bad, so I made this PR to log when this happens: https://github.com/qtile/qtile/pull/4041
